packages/design-system/.storybook/docs/Grid.tsx

```diff
@@ -1,13 +1,12 @@
 import styled from 'styled-components';
-
-import tokens from '../tokens';
+import tokens from '@talend/design-tokens';
 
 const Grid = styled.div<{ columns?: number }>`
 	display: grid;
 	grid-template-columns: repeat(${({ columns = 3 }) => columns}, minmax(0, 1fr));
 	gap: 2.5rem 5rem;
 
-	@media only screen and (max-width: ${tokens.breakpoints.m}) {
+	@media only screen and (max-width: ${tokens.coralBreakpointM}) {
```

Comment:
This won't work. We still need to change tokens here to be strings rather than CSS custom properties for breakpoints :(